src/elements/content-sidebar/__tests__/Sidebar-test.js

```diff
@@ -128,6 +136,91 @@ describe('elements/content-sidebar/Sidebar', () => {
         });
     });
 
+    describe('onKeyDown', () => {
+        const tabList = [
+            `/${SIDEBAR_VIEW_ACTIVITY}`,
+            `/${SIDEBAR_VIEW_DETAILS}`,
+            `/${SIDEBAR_VIEW_SKILLS}`,
+            `/${SIDEBAR_VIEW_METADATA}`,
+        ];
+
+        test('should naviagate to previous tab', () => {
+            const history = createMemoryHistory();
+            history.push(`/${SIDEBAR_VIEW_DETAILS}`);
+            const wrapper = getWrapper({ history });
+            const instance = wrapper.instance();
+            const onKeyDownWithList = instance.onKeyDown(tabList);
+
+            const event = {
+                nativeEvent: {
+                    stopImmediatePropagation: jest.fn(),
+                },
+                key: KEYS.arrowUp,
+                stopPropagation: jest.fn(),
+                preventDefault: jest.fn(),
+            };
+            onKeyDownWithList(event);
+
+            expect(instance.props.history.location.pathname).toBe(`/${SIDEBAR_VIEW_ACTIVITY}`);
+        });
+
+        test('should naviagate to next tab', () => {
+            const history = createMemoryHistory();
+            history.push(`/${SIDEBAR_VIEW_DETAILS}`);
+            const wrapper = getWrapper({ history });
+            const instance = wrapper.instance();
+            const onKeyDownWithList = instance.onKeyDown(tabList);
+
+            const event = {
+                nativeEvent: {
+                    stopImmediatePropagation: jest.fn(),
+                },
+                key: KEYS.arrowDown,
+                stopPropagation: jest.fn(),
+                preventDefault: jest.fn(),
+            };
+            onKeyDownWithList(event);
+
+            expect(instance.props.history.location.pathname).toBe(`/${SIDEBAR_VIEW_SKILLS}`);
+        });
+
+        test('should stay in the same tab if key is not up nor down', () => {
```

Comment:
Minor: This should say "**neither** up nor down"

Reply:
Thanks 😂